packages/auth/src/index.ts

```diff
@@ -0,0 +1,34 @@
+export async function createSpeechmaticsJWT({
+  type,
+  apiKey,
+  clientRef,
+  ttl = 60,
+  managementPlatformURL = 'https://mp.speechmatics.com/v1',
+}: {
+  type: 'batch' | 'rt' | 'flow';
+  apiKey: string;
+  clientRef?: string;
+  ttl?: number;
+  managementPlatformURL?: string;
+}): Promise<string> {
+  if (type === 'batch' && !clientRef) {
+    throw new Error(
+      'Must set the `client_ref` parameter when using temporary keys for batch transcription. See documentation at https://docs.speechmatics.com/introduction/authentication#batch-transcription',
```

Comment:
Do we want to enforce `client_ref` for batch jwt's?

Reply:
I brought this up in the SE channel to get their thoughts. It seems there might be one customer who is using JWTs without client_ref, but I'm unclear about the use-case.

I'm still inclined to enforce it here as a sane default. Worst case if they really need an escape hatch, they can always call the endpoint directly.

Reply:
Sounds good to me